src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -99,6 +87,29 @@
 	protected static Properties appExtSecurePropertiesFile=null;
 	protected static Properties appExtPropertiesFile=null;	
 
+	private static final HashMap<String,String> apiListRegexWithDebugMsg = new HashMap<String,String>(){
+		private static final long serialVersionUID = -1389517682290891890L;
```

Comment:
If I don't have the serialization I receive the following error:
"The serializable class  does not declare a static final serialVersionUID field of type long"